tests/sql/operators/test_load_file.py

```diff
@@ -66,6 +67,19 @@ def test_load_file_with_http_path_file(sample_dag, database_table_fixture):
     assert df.shape == (3, 9)
 
 
+@pytest.mark.integration
+@mock.patch.dict(os.environ, {"ASTRO__DATAFRAME__ALLOW_UNSAFE_STORAGE": "False"})
+def test_unsafe_loading_of_dataframe(sample_dag):
+    with pytest.raises(AirflowException):
+        with sample_dag:
+            load_file(
+                input_file=File(
+                    "https://raw.githubusercontent.com/astronomer/astro-sdk/main/tests/data/homes_main.csv"
+                ),
+            )
+        test_utils.run_dag(sample_dag)
```

Comment:
Should we also have some assert to confirm that we logged the desired message to the end-user? Even a subset of it would be great!

Reply:
@tatiana we can make a custom exception and ensure that the custom exception is thrown. This PR was more to get sign off that this is a good solution.

Reply:
You can pass a "match" kwarg to `pytest.raises`. Example: https://stackoverflow.com/a/55982417 or https://docs.pytest.org/en/6.2.x/assert.html#assertions-about-expected-exceptions

Definition: https://docs.pytest.org/en/6.2.x/reference.html#pytest.raises